manifests/init.pp

```diff
@@ -270,6 +278,10 @@
   Hash $bootstrap_users = {},
   Variant[Undef,String] $bootstrap_post_bootstrap = undef,
   Variant[Undef,String] $bootstrap_post_init = undef,
+  Boolean $is_standby = false,
+  String $remote_host = undef,
```

Comment:
The parameter names should align with the configuration.

```
standby_cluster_host
standby_cluster_port
standby_cluster_primary_slot_name
```

Reply:
yes but we use remote server configuration params to configure standby cluster so I preferred to use remote names,Will make requested changes .